sdk/javascript/test/facade/NemFacade_spec.js

```diff
@@ -95,6 +95,30 @@ describe('NEM Facade', () => {
 
 	// endregion
 
+	// region now
+
+	it('can create current timestamp for network via now', () => {
+		for (;;) {
+			// Arrange: affinitize test to run so that whole test runs within the context of the same millisecond
+			const startTime = new Date().getTime();
+			const facade = new NemFacade('testnet');
+
+			// Act:
+			const nowFromFacade = facade.now();
+			const nowFromNetwork = facade.network.fromDatetime(new Date(Date.now()));
+
+			const endTime = new Date().getTime();
+			if (startTime !== endTime)
+				continue; // eslint-disable-line no-continue
```

Comment:
Should there be a max try if not able to run tests in the same millisecond?

Reply:
can add one if you want, but it would have to be a really slow machine for it not to execute in a ms so was unsure if needed